src/components/PurchaseOrder/PO.js

```diff
@@ -362,7 +369,13 @@ class PO extends Component {
       </PaneMenu>
     );
 
-    if (!order) {
+    if (hasError && !this.hasError) {
+      showToast('ui-orders.errors.orderNotLoaded', 'error');
```

Comment:
I was thinking you just call `onClose` after it, to return on list

Reply:
Yes, we can, but when we use onClose, we have a problem with opening another order, because the resources was not changed, and we again call onClose and show toast.